frameioclient/client.py

```diff
@@ -396,6 +396,25 @@ def update_comment(self, comment_id, **kwargs):
     endpoint = '/comments/{}'.format(comment_id)
     return self._api_call('post', endpoint, payload=kwargs)
 
+  def reply_comment(self, comment_id, **kwargs):
+    """
+    Reply to an existing comment.
```

Comment:
I wonder if we'd want to tell us user in the docstring that (as I just learned today) comments can't be posted to a version stack without grabbing the `cover_asset_id`

Reply:
@jhodges10 Thoughts on this? I can write the message if you're into it.